src/method_jd.jl

```diff
@@ -257,18 +250,13 @@ function jd_effenberger(::Type{T},
     end
 
     conveig += 1
-    Λ[1,1] = λ
-    X[:,1] = u
-    deflated_nep = DeflatedGenericNEP(nep, Λ[1:conveig,1:conveig], X[:,1:conveig])
+    deflated_nep = deflate_eigpair(nep, λ, u)
```

Comment:
Would it be possible to pass the deflation `mode` parameter as an optional parameter to the JD-call?